app/src/main/java/org/p2p/wallet/notification/AppNotificationManager.kt

```diff
@@ -35,7 +36,7 @@ class AppNotificationManager(
         )
     }
 
-    private val notificationManager = context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+    private val notificationManager = context.getSystemService<NotificationManager>()!!
```

Comment:
Are you sure that it cannot be nullable ?

Reply:
context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager 
== 
context.getSystemService<NotificationManager>()!!

these lines throw NPE nevertheless